Sources/RawStream.swift

```diff
@@ -56,13 +56,6 @@ extension _StreamType {
 /// Represents an underlying stream generalized over EventType and used by
 /// higher-level implementations like `Stream<T>` or `Operation<T, E>`.
 public protocol RawStreamType: _StreamType {
-
-  /// The type of events generated by the stream.
-  associatedtype Event: EventType
-
-  /// Register an observer that will receive events from a stream.
-  @warn_unused_result
-  func observe(observer: Event -> Void) -> Disposable
```

Comment:
I had a problem with this not being redeclared here, but it seems to be working now.